src/CommandLineUtils/HelpText/DefaultHelpTextGenerator.cs

```diff
@@ -33,13 +34,18 @@ public class DefaultHelpTextGenerator : IHelpTextGenerator
         /// <summary>
         /// Initializes a new instance of <see cref="DefaultHelpTextGenerator"/>.
         /// </summary>
-        protected DefaultHelpTextGenerator() { }
+        public DefaultHelpTextGenerator() { }
 
         /// <summary>
         /// Determines if commands are ordered by name in generated help text
         /// </summary>
         public bool SortCommandsByName { get; set; } = true;
 
+        /// <summary>
+        /// If set it overrides, the console width is not from the running system.
+        /// </summary>
+        public int? OverriddenConsoleWidth { get; set; } = null;
```

Comment:
```suggestion
        public int? MaxLineLength { get; set; } = null;
```

Can we rename this for consistency with the API naming used elsewhere?